docs/build/build-with-thirdweb/one-click-deploy.md

```diff
@@ -56,37 +59,40 @@ By the end of this tutorial, you will:
 1. Clone the repository: 
  
    ```sh
-   git clone https://github.com/celo-org/one-click-quickstart
-   cd one-click-quickstart
+   git https://github.com/atejada/celo-one-click-deploy
+   cd celo-one-click-deploy
    ```
-   
+  
 2. Install dependencies:
 
 
-	```sh
-	npm install
-	```
+   ```sh
+   npm install
+   ```
 	
 3. Create a .env file with the following content:
 
 
-	```sh
-	VITE_CLIENTID=your_thirdweb_client_id
-	VITE_ADDRESS=your_nft_contract_address
-	VITE_ADRESS_TO=your_wallet_address
-	```	
+   ```sh
+   VITE_CLIENTID = THIRD_WEB_CLIENT_ID
+   VITE_ADDRESS = MINTABLE_NFT_CONTRACT
+   VITE_GALLERY_ADDRESS = NFT_GALLERY_CONTRACT
+   VITE_ADDRESS_TO = ACCOUNT_TO_SEND_CELO_TO
+   ```	
 	
 4. Run the project:
 
 
-	```sh
+   ```sh
    npm run dev
-	```
+   ```
 	
-Once the project is running, there will be two links, the first named **Send Celo** and the second named **Mint NFT**. 
+Once the project is running, there will be three links, the first named **Send Celo**, the second **NFT Gallery** and the third 
+named **Mint NFT**.
+
 The first one will be displayed by default. Click on the **Connect** button to connect the wallet. Enter an 
-address and an amount of Celo to transfer. The second link will display an NFT along with its description and by pressing
-the mint button, 0.1 Celo will be paid.
+address and an amount of Celo to transfer. The second link will display an NTF with a combo box at the top, to choose between 3 different
+NFTs. The third and last link will display an NFT along with its description and by pressing the mint button, 0.1 Celo will be paid.
 
 ## Join Build with Celo - Proof of Ship
 
```

Comment:
this whole section needs to be updated. It should just link to this page with a call to action to build your project on Celo

https://celoplatform.notion.site/Build-With-Celo-Proof-of-Ship-17cd5cb803de8060ba10d22a72b549f8

Reply:
Done. Although we would need to change the repo from mine to a Celo one